### ISSUE TITLE
Timestamp issue with _airbyte_emitted_at and _airbyte_normalized_at

### ISSUE THREAD

**Issue by ltw94** [airbytehq/airbyte#38463]
### Topic

Timestamp inconsistency in normalization

### Relevant information

Hi,
There is an inconsistency in the timestamps used for the columns _airbyte_emitted_at, _airbyte_normalized_at generated in the normalization process. The destination is Snowflake and the local timezone for our snowfalke account is Madrid (UTC +2 in summer).

I performed a sync at 2024-05-21 12:28 Madrid local time and I got the following values:
* _airbyte_emitted_at: 2024-05-21 10:28:49.220 +0200
* _airbyte_normalized_at: 2024-05-21 10:34:41.501 +0000

In summary, the _airbyte_normalized_at value is correct (since 10:34 +000 is the same as 12:34 +0200) but _airbyte_emitted_at is incorrect; it should be 12:28 +0200.

Can you investigate this inconsistency in the timestamps? thanks

**Comment by marcosmarxm** [comment]
Hello @ltw94 thanks for reporting the issue but normalization is deprecated and won't receive updates anymore. Please upgrade your destination to latest version to use Dv2 (typing-deduping) method.
